apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -372,68 +409,83 @@ def grade(element_html, data):
 
     # Get true answer (if it does not exist, create no grade - leave it
     # up to the question code)
-    a_tru = pl.from_json(data["correct_answers"].get(name, None))
+    a_tru: Any = pl.from_json(data["correct_answers"].get(name))
     if a_tru is None:
         return
 
     # Get submitted answer (if it does not exist, score is zero)
-    a_sub = data["submitted_answers"].get(name, None)
-    if a_sub is None:
-        data["partial_scores"][name] = {"score": 0, "weight": weight}
-        return
-    # If submitted answer is in a format generated by pl.to_json, convert it
-    # back to a standard type (otherwise, do nothing)
-    a_sub = pl.from_json(a_sub)
-
-    # Cast both submitted and true answers as np.float64, because...
-    #
-    #   If the method of comparison is relabs (i.e., using relative and
-    #   absolute tolerance) then np.allclose is applied to check if the
-    #   submitted and true answers are the same. If either answer is an
-    #   integer outside the range of int64...
-    #
-    #       https://docs.scipy.org/doc/numpy-1.13.0/user/basics.types.html
-    #
-    #   ...then numpy throws this error:
-    #
-    #       TypeError: ufunc 'isfinite' not supported for the input types, and
-    #       the inputs could not be safely coerced to any supported types
-    #       according to the casting rule ''safe''
-    #
-    #   Casting as np.float64 avoids this error. This is reasonable in any case,
-    #   because <pl-number-input> accepts double-precision floats, not ints.
-    #
-    if np.iscomplexobj(a_sub) or np.iscomplexobj(a_tru):
-        a_sub = np.complex128(a_sub)
-        a_tru = np.complex128(a_tru)
-    else:
-        a_sub = np.float64(a_sub)
-        a_tru = np.float64(a_tru)
+    # a_sub = data["submitted_answers"].get(name, None)
```

Comment:
Remove commented out code.

Reply:
Removed as suggested; IIRC and for future reference, those were a leftover from when we weren't sure whether we have to check for blank submissions in grading since we weren't sure whether blank submissions would be caught in `parse()`, but as it turns out, it is caught in `parse()`.